### PR TITLE
MLPAB-1104 Add trust corporation conduct content

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: app/internal/notify/client.go [hunk 1]
```diff
@@ -13,18 +13,72 @@ import (
...
+		CertificateProviderPaperLpaDetailsChangedSMS:             "94477364-281a-4032-9a88-b215f969cd12",
+		CertificateProviderPaperMeetingPromptSMS:                 "0eba4e55-c07e-4427-b4ad-b03e08dad8ca",
+		CertificateProviderReturnEmail:                           "dd864a1a-64b4-4b4e-b810-86267ebd6476",
+		ReplacementAttorneyInviteEmail:                           "bf79859b-72b7-4701-bfd3-22ac6f0908c8",
+		ReplacementTrustCorporationInviteEmail:                   "bf79859b-72b7-4701-bfd3-22ac6f0908c8",
+		SignatureCodeEmail:                                       "7e8564a0-2635-4f61-9155-0166ddbe5607",
+		SignatureCodeSMS:                                         "dfa15e16-1f23-494a-bffb-a475513df6cc",
+		TrustCorporationInviteEmail:                              "9be88a99-21c0-4808-8d6a-52af366e44aa",
+	}
+)
```
Comment (hawx) [inline app/internal/notify/client.go:68]: very slightly better?

File: app/internal/page/share_code.go [hunk 2]
```diff
@@ -69,6 +66,13 @@ func (s *ShareCodeSender) SendCertificateProvider(ctx context.Context, template
 }
 
 func (s *ShareCodeSender) SendAttorneys(ctx context.Context, appData AppData, lpa *Lpa) error {
+	if err := s.sendTrustCorporation(ctx, notify.TrustCorporationInviteEmail, appData, lpa, lpa.Attorneys.TrustCorporation, false); err != nil {
+		return err
+	}
+	if err := s.sendTrustCorporation(ctx, notify.ReplacementTrustCorporationInviteEmail, appData, lpa, lpa.ReplacementAttorneys.TrustCorporation, true); err != nil {
```
Comment (hawx) [inline app/internal/page/share_code.go:72]: I'm using the same templates at the moment, but I'm sure they will be different when these emails are looked at

File: app/internal/notify/client.go (hunk 1 above)
Comment (acsauk) [reply app/internal/notify/client.go:68]: I'll take it 😁

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**